### PR TITLE
Exclude files in production

### PR DESCRIPTION
**Pull request by Spomky**
| Q             | A
| ------------- | ---
| Bug fix?      | no 
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| Tickets       | no
| License       | MIT
| Doc PR        | no

With this .gitattributes files, unwanted files/folders in production are not installed.
By default, I propose you to keep only the library and template files.
Feel free to ask if you have any reason to keep some of them.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,14 @@
+bin export-ignore
```
Comment (dunglas) [inline .gitattributes:1]: The `bin/schema` file must be exported. It's the entrypoint of the tool.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+data export-ignore
+tests export-ignore
+.editorconfig export-ignore
+.php_cs.dist export-ignore
+.travis.yml export-ignore
+box.json.dist export-ignore
+CHANGELOG.md export-ignore
```
Comment (dunglas) [inline .gitattributes:8]: I would keep this one.

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+composer.lock export-ignore
+CONTRIBUTING.md export-ignore
+phpstan.neon export-ignore
+phpunit.xml.dist export-ignore
+README.md export-ignore
```
Comment (dunglas) [inline .gitattributes:13]: And this one too!

**Comment by Spomky** [comment]
Hi @dunglas,

Thank you for the review. The PR has been updated accordingly.

BR

**Comment by dunglas** [comment]
Thanks @Spomky!

**Pull request merged**